metricq/agent.py

```diff
@@ -430,7 +418,7 @@ def cleanup() -> None:
                 cleanup()
                 raise te
         elif timeout:
-            self.event_loop.call_later(timeout, cleanup)
+            asyncio.get_running_loop().call_later(timeout, cleanup)
```

Comment:
```suggestion
            self._event_loop.call_later(timeout, cleanup)
```

Reply:
I don't have self.event_loop in that class.